packages/devtools_app/lib/src/preferences.dart

```diff
@@ -6,35 +6,25 @@ import 'package:flutter/foundation.dart';
 
 import 'config_specific/logger/logger.dart';
 import 'globals.dart';
-import 'ui/theme.dart' as devtools_theme;
 
 /// A controller for global application preferences.
 class PreferencesController {
-  PreferencesController() {
-    _init();
-  }
-
-  final ValueNotifier<bool> _darkModeTheme =
-      // ignore: deprecated_member_use_from_same_package
-      ValueNotifier(devtools_theme.isDarkTheme);
+  final ValueNotifier<bool> _darkModeTheme = ValueNotifier(true);
```

Comment:
this is now the single source of truth for the color scheme instead of referencing the deprecated source of truth.